docs/settings/alert-action-settings.asciidoc

```diff
@@ -227,6 +227,8 @@ Sets the number of actions that will run ephemerally. To use this, enable
 ephemeral tasks in task manager first with
 <<task-manager-settings,`xpack.task_manager.ephemeral_tasks.enabled`>>
 
+deprecated::[8.8.0]
```

Comment:
This kind of makes it look like it applies to the setting below it, but maybe that is just me 🙈

Reply:
I see a few other places use `Deprecated.` at the beginning of the paragraph. Maybe that would solve the issue. @lcawl wdyt?

Reply:
I'll add a commit to make it look more like the ones in https://www.elastic.co/guide/en/kibana/master/monitoring-settings-kb.html